functions-iterators/starter.js

```diff
@@ -1,82 +1,236 @@
-/* ============ BEFORE YOU GET STARTED ============ */
+// //PART0://
+var name = "Jumanah"
 
-/* here is a very short function that pops up an alert window that says 'Hello!' */
-function hello() {
-  alert('Hello!');
+function justDoIt(name) {
+    return name + " " + "JUST DO IT !!";
 }
 
-/* Here's the steps to test it:
+// console.log(justDoIt(name));
 
-- Open index.html in chrome.
-- Type the keyboard shortcut `cmd + option + j`. Dev tools should open to the console.
-- You should see the message saying 'starter.js is connected!'
-- Underneath that, next to the blue arrow, type `hello()` to call the `hello` function
-- You should get an alert window that says HELLO!
 
-*/
+// //PART1://
 
-/* ================= AS YOU COMPLETE EACH STEP ============= */
 
-/* After each step, perform the following commands in the terminal:
-1. git add .
-2. git commit -m "add completed part PART-NUMBER-HERE"
-This will help us track your progress and it will give you practice writing good commit messages!! 
-*/
+function bigOrSmallString(bigOrS) {
+    if (bigOrS.length >= 10) {
+
+        return (`${bigOrS} "This word is loooooong!"`);
+    } else {
+        return (`${bigOrS}"this is short"`);
+    }
+}
+//  console.log(bigOrSmallString('hlolalala'));
 
-/* ++++====+++==++++==+++++======++++===++++++=====+++==++++++++++===== */
-/* ======================= WRITE YOUR ANSWERS HERE!!!!! =============== */
-/* ++++====+++==++++==+++++======++++===++++++=====+++==++++++++++===== */
 
-// PART 0: Just DO it!!
 
+// // PART2 //
 
 
-// PART 1: Big or Small String?
 
+function oddOrEvenString(wordLength) {
+    if (wordLength.length() % 2 === 0) {
```

Comment:
to access the length property only `length`, not `length()` it is not a function